plugins/calendar/cmd/argParser.go

```diff
@@ -0,0 +1,75 @@
+package cmd
```

Comment:
A folder/package named `cmd` is usually associated with the command line handler, would be better to move this file to a package inside the `pkg` folder, like `plugins/calendar/pkg/parser/argParser.go` and rename the package to `parser`